### DISCUSSION TITLE
Problem with character encoding

### DISCUSSION THREAD

**Discussion by Dorota-Toskania** [community/community#37324] [Other Feature Feedback, Questions, & Ideas]
Can anyone help me?
On my private profile I recognized a character encoding problem in old projects. Everywhere.
Inside Markdown code and inside jupyter notebooks. Like on the screens.

I'm using a corporate Github account on the same computer and browser, and I'm fine.

Thank You 

![github2](https://user-images.githubusercontent.com/68709439/198000764-227895db-daf9-4329-b03e-563aafbb6424.PNG)
![github](https://user-images.githubusercontent.com/68709439/198000765-4cebb8f6-4f6e-411d-8c57-48a017ac2976.PNG)

**Comment by airtower-luna** [comment]
Check the character encoding used to save those files. Are they in UTF-8, or any legacy encoding? In the latter case, try converting them.

**Reply by Dorota-Toskania** [reply]
these files have been in Github for many months and displayed correctly. Only today I saw some coding problem. Problem also applies to other projects that are not mine, e.g.
![Github3](https://user-images.githubusercontent.com/68709439/198004909-18d61fa8-d40c-4bb0-8fad-288d717570db.PNG)

**Reply by airtower-luna** [reply]
In that case you might want to [send a bug report](https://support.github.com/contact/bug-report) to GitHub support, so they can take a look. Whether the files are UTF-8 could still be important.

**Reply by Dorota-Toskania** [reply]
Tahnk you for your help
